start/sw/web-app/about.md

```diff
@@ -1,10 +1,31 @@
 ---
-label: need-help
-
 title: Web Application
 subsection: web-app
 section: start-sw
-description:
+description: Short introduction to web application development
 ---
 
-# Web Application
+# Web Application Development
+
+Web application development is a broad topic, which covers range of different technologies. Under web application, we can imagine anything from classic server-side apps that render dynamic HTML pages, to real-time audio/video apps which allow us to watch movies or videochat online. Even if we only focus on the applications that users display in the web browser, a full-stack web developer needs to know dozens of technologies. What is common for all of them though, Fedora is here to offer all needed components and great development environment.
+
+For learning more about web technologies, visit for example web [w3schools.com](https://www.w3schools.com/).
+
+## Technologies used for web development
+
+One technology that is almost always needed is HTML, a markup language for describing rich documents, which is almost always combined with CSS to make the HTML elements to look nice. To make the web applications more dynamic in the browser, JavaScript is used, often through some framework or libraries like JQuery or React. Even for the CSS styling, there are libraries to make the web applications look better and not reimplement all from scratch. One great example of such library is [patternfly](https://www.patternfly.org/).
```

Comment:
Please shorten and nest into paragraps with headings like so:

```
### HTML
```

So people may actually skip what they're familiar with. What we're trying to do is give jump-start to developers. If you want, you can have more advanced pages for the specific topic- which would be great actually!

It can be short, don't worry; f.e.:
https://developer.fedoraproject.org/tools/vagrant/vagrant-virtualbox.html

Reply:
Instead of using more paragraphs, I've shortened it and used a list, so it should be more readable now.